src/UniGetUI/Pages/DialogPages/DesktopShortcuts.xaml

```diff
@@ -76,14 +76,14 @@
                         <DataTemplate x:DataType="local:ShortcutEntry">
                             <Grid ColumnSpacing="4">
                                 <Grid.ColumnDefinitions>
-                                    <ColumnDefinition Width="Auto"/>
-                                    <ColumnDefinition Width="Auto"/>
+                                    <ColumnDefinition Width="50"/>
+                                    <ColumnDefinition Width="24"/>
                                     <ColumnDefinition Width="*"/>
                                     <ColumnDefinition Width="Auto"/>
                                     <ColumnDefinition Width="Auto"/>
                                 </Grid.ColumnDefinitions>
-                                <CheckBox IsChecked="{x:Bind Enabled}" Checked="{x:Bind Enable}" Unchecked="{x:Bind Disable}" Grid.Column="0"/>
-                                <widgets:LocalIcon Icon="localpc" Grid.Column="1" Height="24" Width="24"/>
+                                <CheckBox IsChecked="{x:Bind IsChecked, Mode=TwoWay}" Grid.Column="0"/>
+                                <widgets:LocalIcon Icon="Launch" Grid.Column="1" Height="24" Width="24"/>
```

Comment:
Why was this icon changed to Launch? The launch icon is already used a couple lines below on the "launch shortcut" button.